pg_search/Cargo.toml

```diff
@@ -33,7 +33,7 @@ libc = "0.2.152"
 memoffset = "0.9.0"
 once_cell = "1.18.0"
 tokenizers = { version = "0.9.1", path = "../tokenizers" }
-pgrx = { git = "https://github.com/paradedb/pgrx.git", rev = "2737a13" }
```

Comment:
Because pgrx `0.12.0` has merged https://github.com/pgcentralfoundation/pgrx/pull/1818, we could safely update now.